public/css/icinga/main.less

```diff
@@ -196,7 +196,7 @@ a:hover > .icon-cancel {
 }
 
 .name-value-table > tbody > tr > th {
-  color: @text-color-light;
+  color: @gray;
```

Comment:
The control-label-group of the object features now use `@text-color-light` instead of `@gray`. This now does the exact opposite, can you please decide which to use in both cases?

Reply:
Changed to `@text-color-light` in both cases with dc5a427.

Also see https://github.com/Icinga/icingadb-web/pull/216/commits/e584755f485e05c298b19725ecd8c148ee637cc5

Reply:
✅